lib/Request.js

```diff
@@ -155,10 +154,10 @@ class Request {
    * Promise).
    */
 
-  each(func, isAsync) {
+  each(func, isAsync, concurrency) {
```

Comment:
Yeah.  If you do something like

```
const ctx = context.concurrency(4);

ctx.forEach(someFn)
  .then(() => ctx.map(anotherFn))
```

the `.map()` would run with a concurrency of `1`, which doesn't quite seem like a reasonable or expected behavior.